src/app/email-validator/email.validator.ts

```diff
@@ -1,17 +1,16 @@
-import {Control} from '@angular/common';
-interface IValidationResult {
-  [key: string]: boolean;
-}
+import {FormControl} from "@angular/forms";
 
 export class EmailValidator {
-  static validEmail(control: Control): IValidationResult {
```

Comment:
don't we need that IValidationResult interface anymore? Any reason why to remove it?